src/main/java/com/exasol/sql/Table.java

```diff
@@ -2,15 +2,16 @@
 
 import com.exasol.sql.ddl.create.CreateTableVisitor;
 import com.exasol.sql.ddl.drop.DropTableVisitor;
-
-import java.util.Optional;
+import com.exasol.sql.dml.insert.InsertVisitor;
+import com.exasol.sql.dml.merge.MergeVisitor;
+import com.exasol.sql.dql.select.SelectVisitor;
 
 /**
  * This class represents a {@link Table} in an SQL Statement
  */
 public class Table extends AbstractFragment {
     private final String name;
-    private final Optional<String> as;
+    private final String as;
```

Comment:
The name is suspicious

Reply:
I had to find a comment to figure out what it means

Reply:
Changed to `alias` for more clarity.

Reply:
Also adapted the getter.